src/main/resources/META-INF/import.sql

```diff
@@ -0,0 +1,22 @@
+-- PackageType info
+insert into PACKAGE_TYPE (ID, TITLE, DESCRIPTION) values(1, 'Document', 'Send a document');
+insert into PACKAGE_TYPE (ID, TITLE, DESCRIPTION) values(2, 'Package', 'Send a regular package');
+
+-- Address info
+insert into ADDRESS (ID, CITY, NUMBER, STREET) values (1, 'Vilnius', 32, 'Antakalnio g.');
+
+-- PackageSize info
+insert into PACKAGE_SIZE (ID, TITLE, HEIGHT, LENGTH, WIDTH, MAX_WEIGHT) values (1, 'S', 450, 350, 160, 2000);
+insert into PACKAGE_SIZE (ID, TITLE, HEIGHT, LENGTH, WIDTH, MAX_WEIGHT) values (2, 'M', 610, 460, 460, 20000);
+insert into PACKAGE_SIZE (ID, TITLE, HEIGHT, LENGTH, WIDTH, MAX_WEIGHT) values (3, 'L', 1500, 1200, 800, 30000);
```

Comment:
I think we should add XL (bulky one) package size too, which would be way bigger than L.

Reply:
Frontend would need to change stuff then, now we display 3 items for packages.